File: html/Karim/PHP/if-else.php
<?php
// $age=45;


// if($age>=18){
//     echo "You can Vote";
// }
// else{
//     echo "You can not vote";
// }



$a=12;
$b=15;
$c=10;

if($a>$b && $a > $c){
    echo $a  . " is grater than " . $b . " and " . $c;
}
else if($b>$a && $b>$c){
    echo $b  . " is grater than " . $a . " and " . $c;
}
else if($c>$a && $c > $b){
    echo $c  . " is grater than " . $a . " and " . $b;
}
else{
    echo "All Numbers are Equals";
}



?>

File: html/Preeti/if-else.py
# age=19

# if age>=18:
#     print("You can Vote Because You are " + str(age))
# else:
#     print("You can Not Vote Because You are " + str(age))

a=70
b=70
c=70

if a>b and a>c:
    print(str(a) + " is grater than " + str(b) + " and " + str(c))
elif b>a and b>c:
    print(str(b) + " is grater than " + str(a) + " and " + str(c))
elif c>a and c>b:
    print(str(c) + " is grater than " + str(a) + " and " + str(b))
else:
    print("All Numbers are Equals")



a=10
b=20
c=30
d=40
if a>b and a>c and a>d:
    print(str(a)+ "is greater than" + str(b) + "and" + str(c) + "and" + str(d))
elif b>a and b>c and b>d:
    print(str(b) + "is greater than" + str(a) + "and" + str(c) + "and" +str(d))
elif c>a and c>b and c>d:
    print(str(c) + "is greater than"+ str(a) + "and" + str(b) + "and" + str(d))
elif d>a and d>b and d>c:
    print(str(d) + "is greater than" + str(a) + "and" + str(b) + "and" + str(c))
else:
    print("All numbers are equals")
